oci-compute/README.md

```diff
@@ -0,0 +1,239 @@
+oci-compute
+===========
+
+A python script to provision compute instances on [Oracle Cloud Infrastructure](https://www.oracle.com/cloud/) (OCI) using the [Python SDK](https://docs.cloud.oracle.com/en-us/iaas/Content/API/SDKDocs/pythonsdk.htm).
+
+# About
+
+This project illustrates the use of the [OCI Python SDK](https://docs.cloud.oracle.com/en-us/iaas/Content/API/SDKDocs/pythonsdk.htm) to list, provision, start, shutdown and terminate OCI compute instances.
+
+Interaction with OCI from the command line can easily be achieved using the [OCI CLI](https://docs.cloud.oracle.com/iaas/Content/API/Concepts/cliconcepts.htm).
+It is the perfect tool for most tasks, but handling output from the OCI CLI in shell scripts can be cumbersome for complex queries.
+The [OCI Python SDK](https://docs.cloud.oracle.com/en-us/iaas/Content/API/SDKDocs/pythonsdk.htm) offers more flexibility but the API may seem complex at first.
+This project is a practical example on how to use the SDK for managing compute instances.
+
+# Prerequisites
+
+1. A computer running Linux, macOS or Windows.
+1. Python version [supported by the SDK](https://oracle-cloud-infrastructure-python-sdk.readthedocs.io/en/latest/#oracle-cloud-infrastructure-python-sdk-ocisdkversion).
+1. [OCI CLI](https://docs.cloud.oracle.com/iaas/Content/API/SDKDocs/cliinstall.htm) is not required, but will make your setup easier.
+1. A clone of this repository.
+
+# Installation
+
+## Virtual environment
+
+The preferred way of installing the `oci-compute` package is to install it in a [Python virtual environment](https://docs.python.org/3/tutorial/venv.html):
+
+```shell
+# Clone the repository
+git clone https://github.com/oracle/ol-sample-scripts.git
+cd ol-sample-script/oci-compute
+# Create and activate the virtualenv
+python3 -m venv
+source venv/bin/activate
+# Install the package
+pip3 install .
+# Package will be installed as venv/bin/oci-compute
+./venv/bin/oci-compute --version
+```
+
+You don't have _activate_ the virtual environment to run `oci-compute`.
+
+## User level / system wide install
+
+Alternatively you can install the package at user level or system wide. The later is strongly discouraged as it can potentially update system libraries.
```

Comment:
```suggestion
Alternatively you can install the package at user level or system wide. The latter is strongly discouraged as it can potentially update system libraries.
```